brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/PublicationContextMapper.java

```diff
@@ -35,12 +36,35 @@ public static PublicationContext buildPublicationContext(Record record)
         if (isBook(record)) {
             return buildPublicationContextWhenBook(record);
         }
-        if (isReport(record)) {
+        if (isReport(record) || isResearchReport(record)) {
             return buildPublicationContextWhenReport(record);
         }
+        if (isDegree(record)) {
+            return buildPublicationContextWhenDegree(record);
+        }
```

Comment:
lag en custom runtime exception når den ikke kan mappes til noe kontekst istedet for å returnere null.